tox.ini

```diff
@@ -3,20 +3,20 @@ envlist = py27, py27-tornado23, py27-tornado31, docs, flake8
 
 [testenv]
 sitepackages = true
-commands = nosetests -v --with-xunit --xunit-file=xunit-{envname}.xml --with-coverage --cover-package=mopidy
+commands = py.test --junit-xml=xunit-{envname}.xml --cov=mopidy
```

Comment:
There's seems to be some issue with our test suite making `py.test -n 6` for example as slow as or slower than `-n 1`. I'll add pytest-xdist to the deps list in `tox.ini` too, and add `{posargs}` to the command. That way, we can experiment with e.g. `tox -e py27 -- -n 6`. When it works properly, we can make parallel execution the default.